_component_design/choice-tiles.md

```diff
@@ -215,20 +224,36 @@ For selected states, the iconography changes color to match the given state. In
 
 <div class="hxRow" markdown="1">
 
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## States
+
+Choice tiles can exist in an **enabled**, **disabled**, or **error** state.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Multi-select interaction
+### Enabled
```

Comment:
i don't think this visual is accurate (see the opacity changes...)

Reply:
also there are a few tweaks needed to the visual. e.g. 
- icon and option label aren't totally centered in 
the tile. 
- some say "Option Label" some say "Label"  let's standardize on "Option Label"

Reply:
Comment 1: will discuss in bowling on 1/29
Comment 2: Done, roman updated visuals.

Reply:
Removed reference to opacity, done.